test/activerecord_test.rb

```diff
@@ -53,6 +53,8 @@ class User < ActiveRecord::Base
   enumerize :status, :in => { active: 1, blocked: 2 }, scope: true
 
   enumerize :account_type, :in => [:basic, :premium]
+
+  enumerize :relationship, :in => [:single, :married]
```

Comment:
Can you please leave a comment that there is no such column in the migration for purpose?